eq_cir_management_ui/config/config.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+
+class DefaultConfig:
+    """The default application config."""
+
+    # Default timeout for REST requests
+    DEFAULT_TIMEOUT = 5
+
+    SECRET_KEY = os.environ.get("SECRET_KEY", "dev-key-for-development-only")
+    LOG_FORMAT = os.environ.get("LOG_FORMAT", "JSON")
+    LOG_LEVEL = os.environ.get("LOG_LEVEL", "INFO")
+
+    CDN_URL = os.getenv("CDN_URL", "https://cdn.ons.gov.uk")
+
+
+class DeployedConfig(DefaultConfig):
+    """Configuration for the DEV and TEST environment.
+
+    Extends DEFAULT Config.
+    """
+
+    LOG_FORMAT = "JSON"
+
+
+class ProdConfig(DeployedConfig):
+    """Configuration for the UAT and PROD environment.
+
+    Extends DEPLOYED Config.
+    """
+
+
+# Set the config depending on the runtime environment.
+match os.environ.get("RUNTIME_ENVIRONMENT", None):
```

Comment:
This uses the default case, is this still needed?

Reply:
Have refactored this so that it includes future required switching, which can be expanded as system is built